player.cpp

```diff
@@ -584,26 +590,45 @@ void test_player() //!OCLINT tests may be long
     // A players goes ?right/?up upon acceleraton
     {
         player p_forward;
+        coordinate c_before = get_coordinate(p_forward);
         p_forward.accelerate();
```

Comment:
A bit confued here: haven't we agreed that `accelerate()` should *not* move the player?